src/components/flat-table/flat-table-header/flat-table-header.d.ts

```diff
@@ -1,13 +1,16 @@
 import * as React from 'react';
+import { SpacingProps } from '../../../utils/helpers/options-helper';
 
-export interface FlatTableHeaderProps {
+export interface FlatTableHeaderProps extends SpacingProps {
   /** Content alignment */
   align: string;
   children: React.ReactNode | string;
```

Comment:
I realise this isn't related to the changes you've added but children isn't required and doesn't have a default value 
```suggestion
  children?: React.ReactNode | string;
```